src/Colors.js

```diff
@@ -0,0 +1,10 @@
+export const colors2 = {
+    primaryColor: 'rgb(13, 40, 50)',
+    primaryColor2: 'rgb(27,84,106)',
+    footerBackgroundColor: 'rgb(11, 41, 53)',
+    mainGreen: 'rgb(32, 177, 151)',
+    redWarning: 'rgb(156, 57, 53)',
+    white: '#fff',
```

Comment:
There is no special reason, expect if we are using rgba. I will fix that.
Hex is probably better solution, i think.